README.md

```diff
@@ -181,13 +181,23 @@ The actions to wait for. It can be either:
 ##### timeout
 
 Type: `Number`   
-Default: 50
+Default: 2000
 
 The timeout given in milliseconds.
 
+##### throttleDelay
+
+Type: `Number`   
+Default: 0
+
+Specifies the time in milliseconds that every invocation to the action's matcher take place at since the last invocation. When set to zero, throttling is disabled.
+
+When throttling is enabled, the `matcher` will be called at most once per `throttleDelay` milliseconds receiving the array of actions dispatched until that time. If the `matcher` does not resolve the `Promise` until `timeout` milliseconds have elapsed, the `Promise` is rejected throwing `TimeoutError`.
```

Comment:
Looking at the current implementation, having a `0` delay will not necessarily disable throttling but instead it will be clamped to 10ms. This is because internally, lodash is using setTimeout(x, 0) which is always clamped to 10ms in v8 and possible other engines.

Reply:
That is correct. I'm trying to avoid invoking `throttle` conditionally because it will make the coder a bit less tidier. 😄 
So, I had another look at Lodash's docs for `throttle`:

* I was invoking `throttle` incorrectly. 😓The `wait` argument was being passed as an option.
* If we set `leading` option to `true` and `wait` argument to `0`, wouldn't the throttled function be invoked immediately? Although the docs describe that

> If wait is 0 and leading is false, func invocation is deferred until to the next tick, similar to setTimeout with a timeout of 0.

they do not clarify the expected behavior when `leading` is `true`, so I will have to investigate further.

Reply:
I guess Lodash is always calling `setTimeout(fn, 0)` regardless `leading` is `true` or `false`. 😞